README.md

```diff
@@ -3,4 +3,4 @@
 Bindings to [level](https://github.com/Level/level).
 
 ## Examples
-See the [file system example code](https://github.com/grybiena/grybiena.github.io/blob/grybiena/halogen-xterm-example/src/Example/FileSystem.purs) and corresponding [file system example demo](https://grybiena.github.io/halogen-xterm-example/index.html). 
+See the [file system example code](https://github.com/grybiena/grybiena.github.io/blob/grybiena/examples/halogen-xterm/src/Examples/LevelDB/FileSystem.purs) and corresponding [file system example demo](https://grybiena.github.io/examples/halogen-xterm/index.html). 
```

Comment:
Thanks for noticing! I think I have broken the example link again since you opened your PR. [This should work to link to the example](https://grybiena.github.io/example/index.html#xterm).